src/app/tests/suites/certification/Test_TC_SMCO_2_5.yaml

```diff
@@ -0,0 +1,1011 @@
+          PIXIT.SMCO.TEST_EVENT_TRIGGER for Manual Device Mute Test Event Clear"
+      PICS: SMOKECO.S.A0004 && SMOKECO.S.F00 && DGGEN.S.C00.Rsp
+      cluster: "General Diagnostics"
+      endpoint: 0
+      command: "TestEventTrigger"
+      arguments:
+          values:
+              - name: "EnableKey"
+                value: TEST_EVENT_TRIGGER_KEY
+              - name: "EventTrigger"
+                value: TEST_EVENT_TRIGGER_DEVICE_MUTED_CLEAR
+
+    - label:
+          "TH waits for a report of DeviceMuted attribute from DUT with a
+          timeout of 120 seconds"
+      PICS: SMOKECO.S.A0004 && SMOKECO.S.F00
+      command: "waitForReport"
+      attribute: "DeviceMuted"
+      timeout: 120
+      response:
+          value: 0
+          constraints:
+              type: enum8
+
+    - label: "TH reads MuteEnded event from DUT"
+      PICS: SMOKECO.S.A0004 && SMOKECO.S.F00 && SMOKECO.S.E07
+      command: "readEvent"
+      event: "MuteEnded"
+      EVENT_NUMBER: EVENT_NUMBER + 6
+      response:
+          value: {}
+
+    - label:
+          "TH sends TestEventTrigger command to General Diagnostics Cluster on
+          Endpoint 0 with EnableKey field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER for Critical Smoke Alarm Test Event"
+      PICS: SMOKECO.S.A0004 && SMOKECO.S.F00 && DGGEN.S.C00.Rsp
+      cluster: "General Diagnostics"
+      endpoint: 0
+      command: "TestEventTrigger"
+      arguments:
+          values:
+              - name: "EnableKey"
+                value: TEST_EVENT_TRIGGER_KEY
+              - name: "EventTrigger"
+                value: TEST_EVENT_TRIGGER_CRITICAL_SMOKE_ALARM
+
+    - label:
+          "TH waits for a report of SmokeState attribute from DUT with a timeout
+          of 300 seconds"
+      PICS: SMOKECO.S.A0004 && SMOKECO.S.F00 && SMOKECO.S.A0001
+      command: "waitForReport"
+      attribute: "SmokeState"
+      timeout: 300
+      response:
+          value: 2
+          constraints:
+              type: enum8
+
+    - label:
+          "TH sends TestEventTrigger command to General Diagnostics Cluster on
+          Endpoint 0 with EnableKey field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER for Manual Device Mute Test Event"
+      PICS: SMOKECO.S.A0004 && SMOKECO.S.F00 && DGGEN.S.C00.Rsp
+      cluster: "General Diagnostics"
+      endpoint: 0
+      command: "TestEventTrigger"
+      arguments:
+          values:
+              - name: "EnableKey"
+                value: TEST_EVENT_TRIGGER_KEY
+              - name: "EventTrigger"
+                value: TEST_EVENT_TRIGGER_DEVICE_MUTED
+
+    - label: "TH waits 60 Seconds"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 60000
```

Comment:
This is a pretty long time to wait in CI test.  Please either don't turn this on in CI (slow test) or have conditional waits depending on whether we are running in CI (which you can test via PICS).

Reply:
I added PICS_SDK_CI_ONLY, is it ok?

Reply:
> I added PICS_SDK_CI_ONLY, is it ok?

Yes.